### ISSUE TITLE
Feedback for .NET Core page

### ISSUE THREAD

**Issue by akoeplinger** [dotnet/dotnet.github.io#8]
I just browsed the [.NET Core page](http://dotnet.github.io/core/) and here are some feedback items that crossed my mind:
- No "View Source" button. It's very difficult finding a link to the actual github repository, I'd suggest placing a button next to "Getting started" and "About .NET Core"
- The gif that is supposed to show .NET Core in action is **way** too fast, I can't see what's going on. It should also wait a couple of seconds at the end before starting the loop again.
- There's some dithering/quality issues in the gif, which program did you use to capture it? I had very good results with [licecap](http://www.cockos.com/licecap/).
- The Windows install steps mention powershell (PS), but the command only works in CMD, see https://github.com/dotnet/coreclr/issues/1132
- The Mac install steps have a typo: `brew tupdate`

Other than those, the page looks very nice :+1: 

/cc @blackdwarf

**Comment by thesmithee** [comment]
Thanks for the comments @akoeplinger. Some thoughts below.
- Interesting idea. @richlander what do you think?
- I wanted the GIF to be fast just so that people get an idea, then go to "Getting Started" page to get more insight. :)
- I used ffmpeg and Expression Capture to do it. Honestly, I just threw it together really fast, so I'm all up for spiffying it up. Will try licecap to see if it can help me produce a better image quality. 
- Will fix these. Also assigned #1132 over there to myself.

**Comment by akoeplinger** [comment]
>  wanted the GIF to be fast just so that people get an idea, then go to "Getting Started" page to get more insight. :)

That's a good idea in theory, but right now it feels more like this: :laughing:  
![matrix](https://cloud.githubusercontent.com/assets/1376924/8138889/58946a70-114f-11e5-84de-084500ad3e80.gif)

**Comment by thesmithee** [comment]
Well, it would be awesome 90s thingie if that was true. I get your point. Will fix it. :)

**Comment by akoeplinger** [comment]
Ah, one additional thing that I forgot above is that on the getting started page it says:

> It is very easy to get started with Core CLR on your platform of choice.

Core CLR isn't mentioned throughout the other pages, it's .NET Core. We should canonicalize on that.

**Comment by thesmithee** [comment]
Changed that (finally). I think we can close this issue now. @akoeplinger you ok with that?

**Comment by akoeplinger** [comment]
@blackdwarf the only thing I'm still missing is the "View Source" button (or whatever you deem a good fit for linking to the code), but we can address that in a separate issue if you want, so feel free to close this :)
